pep-0550.rst

```diff
@@ -1695,17 +1697,15 @@ Version History
 References
 ==========
 
-.. [1] https://blog.golang.org/context
+.. [1] https://go.dev/blog/context
 
-.. [2] https://msdn.microsoft.com/en-us/library/system.threading.executioncontext.aspx
+.. [2] https://docs.microsoft.com/en-us/dotnet/api/system.threading.executioncontext?redirectedfrom=MSDN&view=net-6.0
```

Comment:
```suggestion
.. [2] https://docs.microsoft.com/en-us/dotnet/api/system.threading.executioncontext
```

These extra query params are just for telemetry or are added by default, so they can be cleaned up